<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>load_file_dialog</class>
 <widget class="QDialog" name="load_file_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load a new file</string>
  </property>
  <widget class="QListWidget" name="fileTypeCanidatesList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="itemNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Load as</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pluginLoaderSelectionView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>92</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>92</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
